atlasdb-config/src/main/java/com/palantir/atlasdb/http/errors/AtlasDbRemoteException.java

```diff
@@ -45,6 +51,19 @@ public String getErrorName() {
         return remoteException.getRemoteException().getErrorName();
     }
 
+    // Cannot be named getStackTrace() because that returns a list of (non-serializable) StackTraceElements.
```

Comment:
Didnt realise the method was on `RuntimeException`, confused it with `RemoteException`.